apps/namada-interface/src/slices/accounts.ts

```diff
@@ -185,6 +185,8 @@ const accountsAtom = (() => {
   );
 })();
 
+const LATEST_SYNCED_HEIGHT_STORAGE = "latest-synced-height"
```

Comment:
I think last block info should be stored in extension local storage. 
https://github.com/anoma/namada-interface/blob/ec3dec8070219f29ccf95e8a50c880da3f032566/apps/extension/src/storage/LocalStorage.ts#L86